server/server.js

```diff
@@ -17,7 +16,7 @@ module.exports = function(port, googleAuthoriser) {
                 console.log("success");
                 adminToken = token;
                 res.cookie("sessionToken", token);
-                res.header("Location", "/dash.html");
+                res.header("Location", "/admin/dash.html");
```

Comment:
i just put it in a different folder to fix the lint problem, stephen's pull request sets it in a different format anyway so it's not gonna be in /admin